### PR TITLE
Added documentation for points in 3D case.

### PR DESCRIPTION
**Pull request by alarshi**
This pull request aims to address the issue  #3599 .
### Before your first pull request:

* [x] I have read the guidelines in our [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) document.

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/master/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/master/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

File: source/boundary_velocity/gplates.cc [hunk 1]
```diff
@@ -866,10 +866,12 @@ namespace aspect
                              "plate reconstruction.");
           prm.declare_entry ("Point one", "1.570796,0.0",
                              Patterns::Anything (),
-                             "Point that determines the plane in which a 2D model lies in. Has to be in the format `a,b' where a and b are theta (polar angle)  and phi in radians.");
+                             "Point that determines the plane in which a 2D model lies in. Has to be in the format `a,b' where a and b are theta (polar angle)  and phi in radians."
+                             "This value is not utilized in the 3D geometry models, and could therefore be set to the default or any user-defined quantity.");
```
Comment (gassmoeller) [inline source/boundary_velocity/gplates.cc:870]:
```suggestion
                             "Point that determines the plane in which a 2D model lies in. Has to be in the format `a,b' where a and b are theta (polar angle)  and phi in radians. "
                             "This value is not utilized in 3D geometries, and can therefore be set to the default or any user-defined quantity.");
```

File: source/boundary_velocity/gplates.cc [hunk 2]
```diff
@@ -866,10 +866,12 @@ namespace aspect
...
           prm.declare_entry ("Point two", "1.570796,1.570796",
                              Patterns::Anything (),
-                             "Point that determines the plane in which a 2D model lies in. Has to be in the format `a,b' where a and b are theta (polar angle)  and phi in radians.");
+                             "Point that determines the plane in which a 2D model lies in. Has to be in the format `a,b' where a and b are theta (polar angle)  and phi in radians."
+                             "This value is not utilized in the 3D geometry models, and could therefore be set to the default or any user-defined quantity.");
```
Comment (gassmoeller) [inline source/boundary_velocity/gplates.cc:874]: same changes as above

File: source/boundary_velocity/gplates.cc (hunk 1 above)
Comment (alarshi) [reply source/boundary_velocity/gplates.cc]: Got it, thank you! :)

**Pull request merged**